app/controllers/api/v2/operatingsystems_controller.rb

```diff
@@ -83,8 +87,15 @@ def bootfiles
 
       private
 
+      def rename_config_template
+        if params[:operatingsystem] && params[:operatingsystem][:config_template_ids].present?
```

Comment:
We probably need to handle `config_template_id` (singular) too for the GET route.